src/plugin/bridge.js

```diff
@@ -1,5 +1,6 @@
 import dgram from 'dgram'
 import { WebSocketServer } from 'ws'
+import Plugin from './plugin'
```

Comment:
Thats a small nitpick: I'd give this import a newline so it is separated from the external dependencies